src/main/java/com/limechain/storage/KVRepository.java

```diff
@@ -36,6 +37,8 @@ public interface KVRepository<K, V> {
      */
     boolean delete(K key);
 
+    List<String> findKeysByPrefix(String prefixSeek, int limit);
```

Comment:
would be consistent to add a javadoc for this function given that all the other have as well